crates/sui-framework/packages/move-stdlib/sources/bit_vector.move

```diff
@@ -3,9 +3,9 @@
 
 module std::bit_vector {
     /// The provided index is out of bounds
-    const EINDEX: u64 = 0x20000;
+    const EIndexOutOfBounds: u64 = 0x20000;
```

Comment:
Small nit, it would have been nice if this was a separate PR :)

Reply:
There will be a follow up with errors worked out in std.